recipe/symfony.php

```diff
@@ -22,17 +26,29 @@
 
 // Assets
 set('assets', ['web/css', 'web/images', 'web/js']);
-// Default true - BC for Symfony < 3.0
-set('dump_assets', true);
+
+// Requires non symfony-core package `kriswallsmith/assetic` to be installed
+set('dump_assets', false);
```

Comment:
@oanhnn Split what into a PR? There isn't really anything that can be separated from *this* PR. Can you elaborate?